### PR TITLE
Feature/custom_dispatchedactionscheduler

### PR DESCRIPTION
**Pull request by akaegi**

### CODE REVIEW

**Comment by akaegi** [comment]
See issue #87

File: ReduxSimple.Tests/EffectsActionOrderTest.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            yield return Reducers.On<IncRandomAction, RootState>(
+                (state, _) => state);
+            
+            yield return Reducers.On<IncRandomFulfilledAction, RootState>(
+                (state, action) => state.WithCounter(action.RandomValue));
+        }
+
+        private Effect<RootState> GetIncRandomActionEffect(ReduxStore<RootState> store) =>
+            Effects.CreateEffect<RootState>(
+                // TODO: Here it would be nice to pass the store as argument... Then we would not need to reside on a shared variable...
```
Comment (Odonno) [inline ReduxSimple.Tests/EffectsActionOrderTest.cs:57]: You should be able to do it, now.

**Review by Odonno — CHANGES_REQUESTED** [review]

File: ReduxSimple.Tests/EffectsActionOrderTest.cs (hunk 1 above)
Comment (akaegi) [reply ReduxSimple.Tests/EffectsActionOrderTest.cs]: Nice 👍 
-> done

**Comment by akaegi** [comment]
Hi @Odonno When is the next release planned? Because I'm waiting for this feature to be integrated into the nuget package ;-)

**Comment by Odonno** [comment]
Oh, I thought I did. I will do it now.

**Comment by Odonno** [comment]
Ok, v3.6.1 will be available soon.

**Comment by akaegi** [comment]
Thanks a bunch :-)

**Pull request merged**